proto/cosmos/nft/v1beta1/nft.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package cosmos.nft.v1beta1;
+
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/nft";
+
+// Class defines the class of the nft type.
+message Class {
+  string id          = 1;
+  string name        = 2;
+  string symbol      = 3;
+  string description = 4;
+  string uri         = 5;
+}
+
+// NFT defines the nft.
+message NFT {
+  string              class_id = 1;
+  string              id       = 2;
+  string              uri      = 3;
+  google.protobuf.Any data     = 10;
```

Comment:
please add comment to the `data` field. Both `data` and `any` has almost no meaning.